### PR TITLE
{math} [foss-2015a] graph-tool (REVIEW) 

### PR DESCRIPTION
**Pull request by hajgato**
needs PR #1585

### CODE REVIEW

**Comment by hajgato** [comment]
Needs PR #1585

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/g/graph-tool/graph-tool-2.2.42-foss-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ computation time) to that of a pure C/C++ library."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+#toolchainopts = {'cstd': 'gnu++11'}
+
+source_urls = ['https://downloads.skewed.de/%(name)s/']
+sources = [SOURCE_TAR_BZ2]
+
+python = 'Python'
+pyver = '2.7.9'
```
Comment (boegel) [inline easybuild/easyconfigs/g/graph-tool/graph-tool-2.2.42-foss-2015a.eb]: use 2.7.10?

**Comment by boegel** [comment]
@hajgato: easyconfig filename for graph-tool is missing the versionsuffix

**Comment by wpoely86** [comment]
ok to test

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 19 out of 19 (3 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/225106813af9e1c29419 for a full test report.

**Comment by wpoely86** [comment]
@boegel ready to merge

**Comment by hajgato** [comment]
@boegel: this seem to be relatively easy to merge.

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (3 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/69c170f7328400a9c968 for a full test report.

File: easybuild/easyconfigs/g/graph-tool/graph-tool-2.2.42-foss-2015a-Python-2.7.10.eb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+ statistical analysis of graphs (a.k.a. networks). Contrary to
+ most other python modules with similar functionality, the core
+ data structures and algorithms are implemented in C++, making
+ extensive use of template metaprogramming, based heavily on
+ the Boost Graph Library. This confers it a level of
+ performance that is comparable (both in memory usage and
+ computation time) to that of a pure C/C++ library."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+#toolchainopts = {'cstd': 'gnu++11'}
```
Comment (boegel) [inline easybuild/easyconfigs/g/graph-tool/graph-tool-2.2.42-foss-2015a-Python-2.7.10.eb]: @hajgato: please remove this?

File: easybuild/easyconfigs/g/graph-tool/graph-tool-2.2.42-foss-2015a-Python-2.7.9.eb [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+ statistical analysis of graphs (a.k.a. networks). Contrary to
+ most other python modules with similar functionality, the core
+ data structures and algorithms are implemented in C++, making
+ extensive use of template metaprogramming, based heavily on
+ the Boost Graph Library. This confers it a level of
+ performance that is comparable (both in memory usage and
+ computation time) to that of a pure C/C++ library."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+#toolchainopts = {'cstd': 'gnu++11'}
```
Comment (boegel) [inline easybuild/easyconfigs/g/graph-tool/graph-tool-2.2.42-foss-2015a-Python-2.7.9.eb]: @hajgato: please remove this?

**Comment by boegel** [comment]
tiny remark left, good to go otherwise

**Comment by hajgato** [comment]
@boegel done!

**Comment by boegel** [comment]
Thanks @hajgato!

**Pull request merged**